src/method.js

```diff
@@ -0,0 +1,79 @@
+class ToDo {
+    static list = [];
+
+    constructor(description, complete = false) {
+      this.description = description;
+      this.complete = complete;
+      this.index = ToDo.list.length;
+      ToDo.list.push(this);
+      this.getList = () => ToDo.list;
+    }
+
+    update() {
+      if (this.complete) {
+        this.complete = false;
+      } else {
+        this.complete = true;
+      }
+    }
+}
+
+// Add items to UI
+const addToList = () => {
+  const todoList = document.getElementById('todo-list');
+  todoList.innerHTML = '';
+
+  ToDo.list.forEach((item) => {
+    const listItem = document.createElement('li');
+    listItem.setAttribute('id', item.index);
+    listItem.classList = 'item-container';
+
+    listItem.innerHTML = `
+      <input class="checkbox" type="checkbox">
+      <input type="text" class="input" value='${item.description}'>
+      <button class="cancel-btn"><i class="fa fa-trash" aria-hidden="true"></i></button>
+      `;
+    todoList.appendChild(listItem);
+
+    const checkbox = listItem.querySelector('input');
+    const text = listItem.querySelector('.input');
+    const textInput = listItem.querySelector('p');
+    const deleteButton = listItem.querySelector('button');
```

Comment:
I think these variables should be defined at the top of the function to make it easy to search or have a place to look  at all the variables